### ISSUE TITLE
FCOS object detection api tensorflow show loss=nan

### ISSUE THREAD

**Issue by nashurgessa** [Stick-To/Object-Detection-API-Tensorflow#56]
FCOS training loss nan, is there a solution for this.
